src/win32/fs.cpp

```diff
@@ -1869,8 +1780,7 @@ bool WinDirAccess::dnext(LocalPath& /*path*/, LocalPath& nameArg, bool /*follows
           || *ffd.cFileName != '.'
           || (ffd.cFileName[1] && ((ffd.cFileName[1] != '.') || ffd.cFileName[2]))))
         {
-            name->assign((char*)ffd.cFileName, sizeof(wchar_t) * wcslen(ffd.cFileName));
-            name->insert(0, globbase);
+            nameArg.localpath.assign(ffd.cFileName, wcslen(ffd.cFileName));
```

Comment:
globbase is no longer used?

Reply:
good spotting - it used to be used in WinDirAccess::dnext, but that seems to have been lost.  Needs investigation.

Reply:
Ok, investigation complete - this was a bug introduced, thanks very much for spotting that!   Fix to restore that functionality pushed.